apps/browser/qml/pages/SettingsPage.qml

```diff
@@ -58,14 +58,14 @@ Page {
                 EnterKey.onClicked: focus = false
             }
 
-            ComboBox {
-                id: searchEngine
+            BrowserListItem {
                 enabled: AccessPolicy.browserEnabled
                 width: parent.width
                 //: Label for combobox that sets search engine used in browser
                 //% "Search with"
                 label: qsTrId("settings_browser-la-search_with")
-
+                iconSource: "image://theme/icon-m-search"
+                value: searchEngineConfig.value
```

Comment:
remove these two line... and push space from left to the context menu background item. 

```
                leftMargin: Theme.horizontalPageMargin + icon.width + Theme.paddingMedium
                Icon {
                    id: icon
                    source: "image://theme/icon-m-search"
                    anchors.verticalCenter: parent.verticalCenter
                    x: Theme.horizontalPageMargin
                }
```

Reply:
@rainemak  Done.